ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -528,7 +528,7 @@ + (void)updateViewController:(UIViewController *)vc
 
     // When backBarButtonItem's title is null, back menu will use value
     // of backButtonTitle
-    [backBarButtonItem setTitle:nil];
+    [backBarButtonItem setTitle:@" "];
```

Comment:
Unfortunately, this change still breaks how the hint for the previous screens looks like. With the changed title, it is being set to empty field, which is unacceptable. Can we think about the better approach or "hack" the hints in other way?

https://github.com/software-mansion/react-native-screens/assets/23281839/5cd8f6c1-0039-4c45-b5de-7ac9c194de8c

Reply:
@dylancom thanks! I've tried some methods mentioned in StackOverflow, but unfortunately setting the text color to `Clear` doesn't fix that bug. Also, I've tried to create a custom menu that will contain the title of the previous screen, but somehow iOS doesn't replace it after calling `setMenu` (I also don't want to steer the menu manually). Hence I'm afraid I won't merge this for now - I don't want to sacrifice one functionality for another, but of course you can patch screens with this change on your own.